src/server/network/regionalRailRoutes.ts

```diff
@@ -48,7 +48,7 @@ const getStationIdsForRoutePatternIds = (tripMap: Record<string, SerializableTri
     const stationIdsMap: Record<string, string[]> = {};
     for (const [routePatternId, trips] of Object.entries(tripMap)) {
         const exemplarTrip = trips
-            .filter((trip) => isRegionalRailTerminus(trip.stopTimes[0].stationId))
+            .filter((trip) => isRegionalRailTerminus(trip.stopTimes[0]?.stationId))
```

Comment:
Just curious what happened here?

Reply:
There was an issue with the latest gtfs bundle and this was my attempt to get it to compile with that data. Let me check, but I think I can probably remove this now as I ended up using a slightly older gtfs bundle (from April) to resolve the issues I was seeing with the latest data.